### ISSUE TITLE
Crash: Incompatibility with LambdAurora/AurorasDecorations

### ISSUE THREAD

**Issue by Oliver-makes-code** [devs-immortal/The-Aether#573]
### What were you doing when the crash occured?

When starting up, [Aurora's Decorations](https://github.com/LambdAurora/AurorasDecorations) has a process where it looks through modded woods, and adds recipes corresponding to them, but it expects it the woods to follow vanilla conventions, as it's to difficult to support individually. Paradise Lost doesn't respect these conventions, causing a crash when starting up. This is not an issue on Aurora's Decoration's side, as it uses vanilla conventions.

### Did it crash the client or server?

Both

### Mod Version

Dev

### Fabric API Version

0.45.2

### Trinkets Version

3.1.0

### Crash Report

```shell
[23:47:29] [Render thread/INFO] (AurorasDeco) [Debug] Applying advancement injection...
[23:47:29] [Render thread/INFO] (AurorasDeco) [Debug] Building 339 advancements...
[23:47:29] [Render thread/ERROR] (AurorasDeco) [AurorasDeco] Could not build recipe aurorasdeco:recipes/building_blocks/woodcutting/the_aether/orange_planks due to a missing item tag the_aether:orange_logs. This probably means the mod the_aether is very likely to break Vanilla's expectations! Please report this issue!
[23:47:29] [Render thread/ERROR] (AurorasDeco) [AurorasDeco] Could not build recipe aurorasdeco:recipes/building_blocks/woodcutting/the_aether/crystal_planks due to a missing item tag the_aether:crystal_logs. This probably means the mod the_aether is very likely to break Vanilla's expectations! Please report this issue!
[23:47:29] [Render thread/ERROR] (AurorasDeco) [AurorasDeco] Could not build recipe aurorasdeco:recipes/building_blocks/woodcutting/the_aether/wisteria_planks due to a missing item tag the_aether:wisteria_logs. This probably means the mod the_aether is very likely to break Vanilla's expectations! Please report this issue!
[23:47:29] [Render thread/ERROR] (AurorasDeco) [AurorasDeco] Could not build recipe aurorasdeco:recipes/building_blocks/woodcutting/the_aether/skyroot_planks due to a missing item tag the_aether:skyroot_logs. This probably means the mod the_aether is very likely to break Vanilla's expectations! Please report this issue!
[23:47:29] [Render thread/ERROR] (AurorasDeco) [AurorasDeco] Could not build recipe aurorasdeco:recipes/building_blocks/woodcutting/the_aether/golden_oak_planks due to a missing item tag the_aether:golden_oak_logs. This probably means the mod the_aether is very likely to break Vanilla's expectations! Please report this issue!
```

### Other mods

```shell
Aurora's Decorations (https://github.com/LambdAurora/AurorasDecorations)
```

**Comment by Jack-Papel** [comment]
Thanks for reporting! Looks like we forgot some item tags.

**Comment by CDAGaming** [comment]
Resolved in the 1.17 branch and 1.18

**Comment by Jack-Papel** [comment]
This has been fixed in 1.6.5+. Thanks for reporting!
